mllib-local/src/main/scala/org/apache/spark/ml/linalg/Matrices.scala

```diff
@@ -999,7 +999,7 @@ object Matrices {
             val data = new ArrayBuffer[(Int, Int, Double)]()
             dnMat.foreachActive { (i, j, v) =>
               if (v != 0.0) {
-                data.append((i, j + startCol, v))
+                data.+=((i, j + startCol, v))
```

Comment:
Writing it as `data += (i, j + startCol, v)` would yield compilation errors:
```
Too many arguments for methods =+(A)
Type mismatch, expected: (Int, Int, Double), actual: Int
```
thus here it written as `data.+=((i, j + startCol, v))`